app/actions.rb

```diff
@@ -2,3 +2,7 @@
 get '/' do
   erb :index
 end
+
```

Comment:
Commented this out since we may have a route that can do this. Will discuss later